frontend/app/src/components/form/schema.ts

```diff
@@ -434,16 +434,16 @@ export const aboutSchema: FormField[] = [
     prepend: 'https://github.com/',
     page: 1
   },
-  {
-    name: 'repos',
-    label: 'Github Repository Links',
-    widget: true,
-    type: 'creatablemultiselect',
-    options: [],
-    note: 'Enter in this format: ownerName/repoName, (e.g. stakwork/sphinx-tribes).',
-    validator: repoArrayStrValidator, // look for 1 slash
-    page: 1
-  },
+  // {
+  //   name: 'repos',
+  //   label: 'Github Repository Links',
+  //   widget: true,
+  //   type: 'creatablemultiselect',
+  //   options: [],
+  //   note: 'Enter in this format: ownerName/repoName, (e.g. stakwork/sphinx-tribes).',
+  //   validator: repoArrayStrValidator, // look for 1 slash
+  //   page: 1
+  // },

```

Comment:
@ecurrencyhodler did we want to remove this Github repo links from the profile, if so I think we should delete this instead of commenting it out

Reply:
@kevkevinpal I am somewhat confused with what type the githubRepositoryLinks fields should be. Could you please help me with it

Reply:
Hey sorry somehow I missed this. Yes let's remove github repo links because we have the github profile.

Reply:
Well right now I have **included** the github repo links section with the **type as text** and **validator-commented out.** 
@ecurrencyhodler @kevkevinpal What is the final version of the about page you guys would want, with the github links section or without it.

Reply:
There are two categories:
Github profile
Github links for a repo

Keep the github profile section but remove the github links for a repo

Reply:
Sure, so completely removing it right. And not commenting it

Reply:
Okay, on it. Thanks. I would remove the other dependencies of it as well right which are in aboutview.tsx